src/sangeeta/CharacterClassPrograms/AsciiOfChar.java

```diff
@@ -0,0 +1,24 @@
+/*Write a method which will return ascii value of given character.
+Hint : Method signature should be int getAsciiOfChar(char ch)
+*/
+package sangeeta.CharacterClassPrograms;
+
+import java.util.Scanner;
+
+public class AsciiOfChar {
+
+	int getAsciiOfChar(char ch) {
+		int temp = (int) ch;
+		System.out.println("ASCII value of character " + ch + " is " + temp);
```

Comment:
You don’t require define int you can direct return ch in method that internally converted